### PR TITLE
Move FrameworkPackage generation to post-signing job

### PR DESCRIPTION
**Pull request by jevansaks**
This change refactors the yaml that was invoking the framework package into a helper that can be invoked from the build (for PR builds) and then from the Nuget generation job (post-signing). 

I had to fix up the MakeFrameworkPackage.ps1 script a bit because it was generating the activatable types list from wrong winmd (the "internal" one, not the sdk directory one), and that was uncovered when running against the artifact drop which only has the sdk one.

### CODE REVIEW

File: build/MUX-Release.yml [hunk 1]
```diff
@@ -70,8 +70,8 @@ jobs:
     inputs: 
       buildType: specific
       buildVersionToDownload: specific
-      project: 'f05f4dbf-3077-427d-91e0-b66d9e4ca374'
-      pipeline: 35915
+      project: $(System.TeamProjectId)
```
Comment (kmahone) [inline build/MUX-Release.yml]: Cool. A lot nicer than including a guid in the file.

File: build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml [hunk 2]
```diff
@@ -19,12 +19,7 @@ steps:
     inputs:
       filename: 'set'
 
-  - task: powershell@2
-    inputs:
-      targetType: filePath
-      filePath: build\Install-WindowsSdkISO.ps1
-      arguments: 18323
-    displayName: 'Install Insider SDK (18323)'
+  - template: MUX-InstallWindowsSDK-Steps.yml
```
Comment (kmahone) [inline build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml]: You're a big fan of yml templates. :-)

File: build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml [hunk 3]
```diff
@@ -47,6 +46,19 @@ jobs:
 
   - template: MUX-InstallNuget-Steps.yml
 
+  - ${{if parameters.signConfig }}:
+    # Install SDK since the custom appx generation needs to resolve winmd types from it.
+    - template: MUX-InstallWindowsSDK-Steps.yml
+
+    # Re-create framework packages because
```
Comment (kmahone) [inline build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: Truncated comment.

File: build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml (hunk 2 above)
Comment (jevansaks) [reply build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml]: They make the templates way nicer to read and manage (IMO). If you disagree, let me know and I'll try to figure out a different strategy. I only wish there was a way to have one library file with sub-templates you could include. Like functions, yknow?

File: build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml (hunk 3 above)
Comment (jevansaks) [reply build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml]: Fixed.

File: build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml (hunk 2 above)
Comment (kmahone) [reply build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml]: I don't disagree.

**Pull request merged**